cogs/misc.py

```diff
@@ -58,11 +58,22 @@ def __init__(self, bot):
             em.add_field(name=u'\U0001F4E5 Msgs received', value=str(self.bot.message_count))
             em.add_field(name=u'\u2757 Mentions', value=str(self.bot.mention_count))
             em.add_field(name=u'\u2694 Servers', value=str(len(self.bot.guilds)))
-            em.add_field(name=u'\ud83d\udcd1 Channels', value=str(channel_count))
+            em.add_field(name=u'\ud83d\udcd1 Channels', value=str(channel_count)+' ('+str(len(self.bot.private_channels))+' private)')
+            em.add_field(name=u'\U0001f46a Users', value=str(len(self.bot.users)))
             em.add_field(name=u'\u270F Keywords logged', value=str(self.bot.keyword_log))
             g = u'\U0001F3AE Game'
-            if '=' in game: g = '\ud83c\udfa5 Stream'
+            if '=' in game:
+                g = '\ud83c\udfa5 Stream'
+                game = game.split('=')
+                game = '[{}]({})'.format(game[0],game[1])
             em.add_field(name=g, value=game)
+            status = str(ctx.message.author.status)
+            if not status == 'offline':
```

Comment:
This looks rather arbitrary - do we know under what circumstances the status variable will give the "correct" value?